tests/api_test.py

```diff
@@ -4264,6 +4264,14 @@ def add(x):
           tracing_add_count += 1
       self.assertEqual(tracing_add_count, 2)
 
+  @parameterized.named_parameters([
```

Comment:
You might consider generalizing this test to capturing values of *all* types?

Reply:
I think `jit` functions for other types have pretty sufficient coverage already... I'm going to leave as-is because I think it makes the intent of the test more clear.